examples/demo_launcher/lib/main.dart

```diff
@@ -177,7 +177,7 @@ class DemoList extends Component {
   Widget build() {
     return new ScrollableList<SkyDemo>(
       items: demos,
-      itemHeight: kCardHeight,
+      itemSize: kCardHeight,
```

Comment:
s/itemSize/itemExtent/

We usually us "Size" to revert to a pair of width and height.  I've been using the term "extent" to mean one dimension that's either width or height...  We can pick another word if you prefer.